swagger/swagger-generator/generator-core/src/main/java/io/servicecomb/swagger/extend/property/creator/PartPropertyCreator.java

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
Please update the license header with the new one.

Reply:
done